SalesforceCommerceCloud/pwa-kit
# [commerce-sdk-react] SCAPI Custom endpoint support

# Description



This PR introduces support for calling SCAPI custom endpoint. It upgrades `commerce-sdk-isormorphic` to the latest version `v2`. Two new hooks are introduced in this PR.

## New hooks
- `useCustomQuery`
- `useCustomMutation`

# Types of Changes





# How to Test-Drive This PR

- pull this branch
- `npm ci`
- `cd ./packages/test-commerce-sdk-react && npm start`
- go to `localhost:3000/custom-endpoint`
- **check the custom query is successfully made with 200 and correct auth token**

# Checklists




## General


## Accessibility Compliance

You must check off all items in **one** of the follow two lists:


_or..._


## Localization
